timesketch/frontend-ng/src/views/Explore.vue

```diff
@@ -586,6 +612,77 @@ export default {
         this.meta.filter_labels.push(label)
       }
     },
+    toggleshowShortcutHelp: function () {
```

Comment:
Since we moved the right side views into the canvas component the explorer component is loaded even when other components like stories, TI or analyzers are visible. To prevent the shortcuts from triggering on the other components I would recommend to wrap them in something like this:
`if (this.currentRouteName === 'Explore') `
Where `currentRouteName` is defined in the computed section:
```
currentRouteName() {
  return this.$route.name
}
```

Reply:
good point, done

Reply:
I would recommend to do this check for all shortcuts and just trigger them on the page that is relevant. 
e.g. at the moment I still can trigger the shortcut help dialog on a story page, but there is actually no shortcut relevant for stories.
I also still can trigger next- & prev-page when looking at a story or graph. And it will paginate in the background even I'm not looking at the table. This is unexpected behaviour for me and I would recommend to make shortcuts only available when someone is viewing the element they are affecting.

Reply:
should be better now, PTAL